src/main/com.tgt.testapp/database/PostgresDAO.java

```diff
@@ -11,18 +11,57 @@
     public static void main(String[] args) {
 
         String url = "jdbc:postgresql://localhost:5432/postgres";
+        String userName = "postgres";
+        String password = "Sadhguru";
```

Comment:
it is not a good practice to hardcode passwords

Reply:
I will figure out how to do it for the next PR